flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecSink.java

```diff
@@ -57,6 +57,13 @@
 @ExecNodeMetadata(
         name = "stream-exec-sink",
         version = 1,
+        consumedOptions = {
+            "table.optimizer.simplify-operator-name-enabled",
+            "table.exec.rank.topn-cache-size",
```

Comment:
These options are clearly wrong. Where is `table.exec.sink.not-null-enforcer` etc?